Content.Client/UserInterface/RoundEndSummaryWindow.cs

```diff
@@ -65,30 +66,40 @@ public RoundEndSummaryWindow(string gm, TimeSpan roundTimeSpan, List<RoundEndPla
             //Create labels for each player info.
             foreach (var plyinfo in manifestSortedList)
             {
-
                 var playerInfoText = new RichTextLabel()
                 {
-                    SizeFlagsVertical = SizeFlags.Fill
+                    SizeFlagsVertical = SizeFlags.Fill,
                 };
 
                 //TODO: On Hover display a popup detailing more play info.
                 //For example: their antag goals and if they completed them sucessfully.
                 var icNameColor = plyinfo.Antag ? "red" : "white";
                 playerInfoText.SetMarkup(
-                    Loc.GetString($"[color=gray]{plyinfo.PlayerOOCName}[/color] was [color={icNameColor}]{plyinfo.PlayerICName}[/color] playing role of [color=orange]{plyinfo.Role}[/color]."));
+                    Loc.GetString("[color=gray]{0}[/color] was [color={1}]{2}[/color] playing role of [color=orange]{3}[/color].",
```

Comment:
This change would lead to `icNameColor` getting translated, which I don't believe is intended.

Reply:
huh i fixed that

Reply:
Nevermind.
![image](https://user-images.githubusercontent.com/10968691/90637899-e06e8800-e22c-11ea-9dbc-a521aa63c583.png)